source/includes/_datasets.md

```diff
@@ -105,12 +96,52 @@ This endpoint lists all the datasets that can be edited by this user.
 Parameter | Default | Description
 --------- | ------- | -----------
 `where` <br> *string* | None | Filter expression used to restrict returned datasets ([ODSQL documentation](https://docs.opendatasoft.com/api/explore/v2.html#where-clause))
-`start` <br> *string* | 0 | Index of the first item to return
+`page` <br> *string* | 1 | Number of the page you want to retrieve.
```

Comment:
Try to avoid using "you" in the documentation. "Number of the page to retrieve" would be better.